recipe/meta.yaml

```diff
@@ -26,51 +26,47 @@ requirements:
   build:
     # pythran code needs clang-cl on windows
     - clang                 # [win]
+    - lld     # [win]
     - {{ compiler("c") }}
     - {{ compiler("cxx") }}
     - llvm-openmp >=4.0.1   # [osx]
     - m2-patch              # [win]
     - patch                 # [not win]
   host:
     - python
-    - cython >=0.29.24,<3.0
+    - cython >=0.29.24
     - llvm-openmp >=4.0.01  # [osx]
-    - numpy   1.19  # [py<310]
-    - numpy   1.21  # [py==310]
-    - numpy   1.23  # [py>=311]
+    - numpy 1.21  # [py<311]
+    - numpy 1.23  # [py>=311]
     - packaging
     - pip
     - pythran
-    - setuptools <=59.4
+    - setuptools
     - wheel
+    - meson-python 0.12.1
```

Comment:
Does not it require >=0.13.0rc0 ? 
https://github.com/scikit-image/scikit-image/blob/v0.20.0/requirements/build.txt#L2

Reply:
yes but it still works with 0.12.1, and 0.13.0 is unreleased

Reply:
Yes, similar to what Charles said, 013.0rc0 is a pre-release version that should not be used as it is an unofficial release. 0.12.1 is the most preferable version to use.